unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/ScenesController/ISelectSceneListener.cs

```diff
@@ -0,0 +1,4 @@
+internal interface ISelectSceneListener
```

Comment:
An `internal` interface seems weird to me, what are the thoughts behind it?

Reply:
Mostly it's just for keeping it inside it's assambly, there is no reason at all to expose that interface to other assemblies. Making it pubic will just generate "intellisense noice" in other files that might reference to this assambly.

Reply:
Fair enough!